packages/wp-now/src/download.ts

```diff
@@ -194,12 +202,32 @@ export async function downloadWordPress(
 }
 
 export async function downloadSqliteIntegrationPlugin() {
-	return downloadFileAndUnzip({
+	// Remove the old SQLite plugin if it exists
+	const oldSqlitePath = path.join(getWpNowPath(), `${SQLITE_FILENAME}-main`);
```

Comment:
Does this download the SQLite integration plugin every time `wp-now` starts? Couldn't it reuse the cached `~/.wp-now` version?

Reply:
No, only if it's missing. 

The code you highlighted only ensures that we remove the old folder. 
The folder changed from `.wp-now` to `.wp-now/mu-plugins`/